features/miniAlert.js

```diff
@@ -0,0 +1,25 @@
+//Thanks Vals
+function inDungeons() {
+    if(!(Scoreboard.getTitle().trim() != "" && Scoreboard.getLines().length > 0)) {

```

Comment:
there should be a space between the if and the (),
yes this is a bit picky but the whole project uses it like that so you should too

Reply:
Didn't Ask.